src/Acuminator/Acuminator.Analyzers/DiagnosticsShortName.resx

```diff
@@ -411,4 +411,7 @@
   <data name="PX1096" xml:space="preserve">
     <value>PXOverrideSignatureMismatch</value>
   </data>
+  <data name="PX1099" xml:space="preserve">
```

Comment:
@EAndrosova please check the short name for PX1099 diagnostic

Reply:
Changed this to avoid noun stacking and therefore ambiguity